### ISSUE TITLE
different input have the same result

### ISSUE THREAD

**Issue by wangpupanjing** [natanielruiz/deep-head-pose#70]
i trained a model use hopenet, but when i test ,every input image have the same result ,why?

**Comment by yafeiliu666** [comment]
hello! I have no the file about face detection, if you have it, please give it me, thank you!

**Comment by howard5758** [comment]
Same problem here. Any updates?

**Comment by ZhongWenY** [comment]
Have you solved this problem? I have encountered the same problem.

**Comment by ZhongWenY** [comment]
> i trained a model use hopenet, but when i test ,every input image have the same result ,why?

Have you solved this problem? I have encountered the same problem.

**Comment by guojilei** [comment]
Continue training，　i have this problem ,  but i continue training  now it is ok ,you can try change your　optimizer and continue training

**Comment by NguyenDangBinh** [comment]
dear @guojilei,
How to train with 300W_LP dataset from at first?

**Comment by LamLuong** [comment]
> Continue training，　i have this problem , but i continue training now it is ok ,you can try change your　optimizer and continue training

How many epochs did you train? and Did you change optimizer or used default of code

**Comment by LamLuong** [comment]
Hi everyone,
I resolved problem above, you should not use default alpha and learning rate value of code, should use this values in paper

**Comment by x1ngzai** [comment]
> Hi everyone,
> I resolved problem above, you should not use default alpha and learning rate value of code, should use this values in paper

hello.Can you send me your final trained model?
Thank you
